tests/utils/test_topk.py

```
@@ -1227,6 +1234,166 @@ def test_algorithms_with_large_k(algo, set_topk_algo):
 
 
+@pytest.mark.parametrize("algo", ["auto", "multi_cta", "filtered"])
+def test_bf16_long_seq_regression_across_algorithms(algo, set_topk_algo):
+    """Regression for bf16 long-seq topk across algorithm overrides."""
+    _require_sm80_for_bf16()
+    if algo == "filtered" and not can_implement_filtered_topk():
+        pytest.skip("GPU does not support filtered topk (requires 128KB shared memory)")
+
+    set_topk_algo(algo)
+
+    logits, _, _, batch_size, _, k, _ = _build_bf16_long_seq_bucket_inputs()
```

Ruff (0.15.1)

[warning] Unpacked variable `batch_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)